package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@whereby.com/browser-sdk",
-    "version": "2.0.0-beta2",
+    "version": "2.0.0-alpha27",
```

Comment:
should this be changing?

Reply:
We are going to switch back to alpha while iterating over a potential solution, to prevent us ending up with dozens of beta releases